dataStructures.f90

```diff
@@ -64,18 +64,26 @@ module date
 
   ! -----------------------------------------------------------------
   ! ???
+  ! TODO: monthList does not account for leap years (second item in list becomes 29)
   subroutine calcDateParameters()
     implicit none
 
-    integer(kind=SI) :: monthList(12)
+    integer(kind=SI) :: monthList(12), i
 
     monthList = [31_SI, 28_SI, 31_SI, 30_SI, 31_SI, 30_SI, 31_SI, 31_SI, 30_SI, 31_SI, 30_SI, 31_SI]
-    ! calculate which day of the year day/month refers to
-    dayOfYear = sum( monthList(1:month - 1_SI) )
+
+    ! calculate day of year from day/month (set in model.parameters)
+    ! January 1 = 0, January 2 = 1, etc...
+    dayOfYear = 0
+    do i=1, (month - 1_SI)
```

Comment:
I don't know :(
I could not get it work using `sum`, for some reason, that's why I changed to a loop.